receiver/statsdreceiver/go.mod

```diff
@@ -3,20 +3,29 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsd
 go 1.18
 
 require (
+	// Note that this Lightstep metrics SDK dependency is temporary.  The same
+	// code is under review upstream, see
+	// https://github.com/open-telemetry/opentelemetry-go/pull/3022
+	github.com/lightstep/otel-launcher-go/lightstep/sdk/metric v1.11.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.59.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.58.0
 	github.com/stretchr/testify v1.8.0
 	go.opencensus.io v0.23.0
 	go.opentelemetry.io/collector v0.59.1-0.20220914173239-9df807d51dad
 	go.opentelemetry.io/collector/pdata v0.59.1-0.20220914173239-9df807d51dad
-	go.opentelemetry.io/otel v1.9.0
+	go.opentelemetry.io/otel v1.10.0
+
+	// Note! This unreleased otel-go dependency (would-be v0.32)
+	// changes histogram inclusivity.
+	// https://github.com/open-telemetry/opentelemetry-go/pull/2982.
```

Comment:
AIU, this change is on the spec but not in any released version (https://github.com/open-telemetry/opentelemetry-specification/commit/ea017156740775ddf132fbd3085cb176101a2042 is only on main). 

On the last SIG, a proposal to block this kind of changes was raised (see #14122); it's fine (IMO, other maintainers opinion may differ) to use specification-conformant code from the SDK that has not yet been released, but it's not fine to if the specification itself has not been released

Reply:
Spec is now released

Reply:
To avoid complications of this nature, I published a copy of the exponential histogram data structure and mapping functions at github.com/lightstep/go-expohisto. This PR is updated to use that repository's v1.0.0 release.